test/GitcoinGovernor.t.sol

```diff
@@ -20,6 +20,7 @@ abstract contract GitcoinGovernorTestHelper is Test, DeployInput {
   address constant TIMELOCK = 0x57a8865cfB1eCEf7253c27da6B4BC3dAEE5Be518;
   address constant PROPOSER = 0xc2E2B715d9e302947Ec7e312fd2384b5a1296099; // kbw.eth
   address constant DEPLOYED_BRAVO_GOVERNOR = 0x1a84384e1f1b12D53E60C8C528178dC87767b488;
+  uint256 constant MAX_REASONABLE_TIME_PERIOD = 302_400; // 6 weeks assuming a 12 second block time
```

Comment:
We added this sanity check to PoolTogether repo and it seemed worthwhile to add them here as well